cmd/zc_traverser_file.go

```diff
@@ -41,6 +41,10 @@ type fileTraverser struct {
 	incrementEnumerationCounter func()
 }
 
+func (t *fileTraverser) isDirectory(isDest bool) bool {
+	return copyHandlerUtil{}.urlIsAzureFileDirectory(t.ctx, t.rawURL, t.p) // This handles all of the fanciness for us.
```

Comment:
it's best to pull the code out of copyHandlerUtil, which we should try to remove towards the end of refactoring. It's a dumping ground for unrelated (sometimes untested) code, best to rid of it for our own good. 😈

Reply:
Done.